packages/console/src/pages/ApiResourceDetails/index.tsx

```diff
@@ -36,6 +39,8 @@ const ApiResourceDetails = () => {
   });
   const [submitting, setSubmitting] = useState(false);
 
+  const [isDeleteOpen, setIsDeleteOpen] = useState(false);
```

Comment:
nit: isDeleteFormOpen better? or maybe  showDeleteForm i.e

Reply:
`isDeleteFormOpen` is better as we will have the mutate-function named `setIsDeleteFormOpen`.
`setShowDeleteFrom` sounds weird.